src/utils/DataBuffer.hxx

```diff
@@ -176,8 +199,13 @@ private:
 
 using DataBufferPtr = std::unique_ptr<DataBuffer, BufferDelete<uint8_t[]>>;
 
+class LinkedDataBufferPtr;
```

Comment:
this declaration seems unnecessary since the class body is just below. Any particular reason this is needed here?

Reply:
removed